src/Compilers/Core/Portable/Symbols/MethodKind.cs

```diff
@@ -15,6 +15,7 @@ public enum MethodKind
         /// An anonymous method or lambda expression
         /// </summary>
         AnonymousFunction = 0,
+
```

Comment:
Consider reverting changes in this file.

Reply:
This does not appear to be addressed. The lack of space was intentional, I believe, to convey the relationship between the two kinds.